content/guides/tasks/update.md

```diff
@@ -0,0 +1,26 @@
+---
+rank: 4
+related_endpoints:
+  - put_tasks_id
+related_guides:
+required_guides: []
+alias_paths: []
+---
+
+# Make changes to an existing task
+
+To update a task in Box you will need to call the
+[`PUT /tasks/:task_id`](e://put_tasks_id) API. Updates to an existing task can
+include `completion_rule`, `message`, `due_at`, and more
+
+<Samples id='put_tasks_id' />
+
+<Message>
+    When a task is created or updated with completion rule all_assignees
+    (default), the task will only be considered completed when all assignees
+    have completed the task. When a task is created or updated with completion
+    rule any_assignee, the task will be considered completed when one assignee
+    has completed the task.
+
+    Value is one of "all_assignees", "any_assignee"
```

Comment:
It's recommended to wrap all fixed values like `all_assignees` and `any_assignee` in backticks.  This a) hides them from the spellchecker (which I think currently fails) and b) marks them as **do not translate** to the translation team. Please do not put quotes inside backticks, e.g. `"all_assignees"`.

Reply:
And yes I should do a style guide on how to do titles, inline code, etc. 😆